src/redux/categories/categories.js

```diff
@@ -0,0 +1,19 @@
+// categories.js
+
+// Actions
+const CHECK_CATEGORIES = 'CHECK_CATEGORIES';
+
+// Reducer
+export default function categoriesReducer(state = {}, action = {}) {
+  switch (action.type) {
```

Comment:
- [ ] The initial state would be an array. Kindly make an variable with an empty array and pass it into your reducer something like this :arrow_down:

```
const initialState = [];
export default function categoriesReducer(state = initialState, action = {}) {
```